core/controlplane/config/templates/cloud-config-controller

```diff
@@ -338,14 +350,16 @@ ssh_authorized_keys:
       content: |
         [Unit]
         Description=Pull and tag a mirror image for pause-amd64
-        After=docker.service
```

Comment:
The start is never called. In logs I can see "loaded" and "enabled" but not even one intent to start it.